components/BlogList/BlogListItem.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import fm from 'front-matter';
+import Error from '../../pages/_error';
+import ImageContainer from '../shared/ImageContainer';
+import { below } from '../../utilities/breakpoint';
+
+const BlogContainer = styled(ImageContainer)`
+  width: 70vw;
```

Comment:
OMG!  Just looked up what `vw` actually did (I know noob), but why aren't we using this more?